python/ray/tune/impl/tuner_internal.py

```diff
@@ -113,6 +114,15 @@ def __init__(
         with open(trainable_ckpt, "wb") as fp:
             pickle.dump(self._trainable, fp)
 
+    def _process_scaling_config(self) -> None:
+        """Converts ``self._param_space["scaling_config"]`` back to a dict so
+        that it can be used to generate search space.
+        """
```

Comment:
nit:
```suggestion
        """Converts ``self._param_space["scaling_config"]`` to a dict so
        that it can be used to generate search space.
        """
```

Also first sentence should be limited to a single line.